server/graphql/models/InviteCode/query.js

```diff
@@ -1,11 +1,9 @@
 import {GraphQLNonNull, GraphQLString} from 'graphql'
 import {GraphQLError} from 'graphql/error'
 
-import {connect} from 'src/db'
+import {InviteCode as ThinkyInviteCode} from 'src/server/services/dataService'
```

Comment:
Same as above; let's not specifically reference Thinky. Use something like `InviteCodeModel`.